pybindsrc/PyBind.cpp

```diff
@@ -0,0 +1,97 @@
+#include "ers/AnyIssue.hpp"
+#include "ers/Issue.hpp"
+#include "ers/IssueFactory.hpp"
+#include "ers/Context.hpp"
+#include "ers/LocalContext.hpp"
+#include "ers/RemoteContext.hpp"
+#include "ers/Severity.hpp"
+#include "ers/ers.hpp"
+
+#include <exception>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+
+PYBIND11_MODULE(_daq_ers_py, liberspy) {
```

Comment:
Similarly, I would use `m` or `module` for the module object rather than `liberspy`

Reply:
Updated in branch